static/elements/chromedash-featurelist.html

```diff
@@ -33,231 +35,280 @@
+            // Do a general search over misc fields.
+            // Remove "=" prefix so "in development"/"proposed" version queries
+            // return results.
+            if (val.indexOf('=') == 0) {
+              val = val.substring(1);
             }
-          });
-        } else { // Do a general search over misc fields.
-          // Remove "=" prefix so "in development"/"proposed" version queries
-          // return results.
-          if (val.indexOf('=') == 0) {
-            val = val.substring(1);
+            var regex = new RegExp(val, 'i');
+            this.filtered = this.features.filter(function (feature, idx, array) {
+              return regex.test(feature.name) || regex.test(feature.category) || regex.test(feature.summary) || regex.test(feature.search_tags) || /*regex.test(feature.shipped_milestone) ||*/
+              regex.test(feature.impl_status_chrome);
+            });
           }
-          var regex = new RegExp(val, 'i');
-          this.filtered = this.features.filter(function(feature, idx, array) {
-            return regex.test(feature.name) || regex.test(feature.category) ||
-                   regex.test(feature.summary) ||
-                   regex.test(feature.search_tags) ||
-                   /*regex.test(feature.shipped_milestone) ||*/
-                   regex.test(feature.impl_status_chrome);
-          });
         }
-      }
-
-      this.filteredChanged();
-
-      return this.filtered.length;
-    },
-    // Returns the closest feature <li> to the scroll top position passed in.
-    featureInView: function(containerScrollTop) {
-      var closest = null;
-      for (var i = 0, li; li = featureLiList_[i]; ++i) {
-        var dist = li.offsetTop - containerScrollTop;
-        if (dist < 0) {
-          dist = -dist * 1.5;
+        this.filteredChanged();
+        return this.filtered.length;
+      },
+      // Returns the closest feature <li> to the scroll top position passed in.
+      featureInView: function(containerScrollTop) {
+        var closest = null;
+        for (var i = 0, li; li = featureLiList_[i]; ++i) {
+          var dist = li.offsetTop - containerScrollTop;
+          if (dist < 0) {
+            dist = -dist * 1.5;
+          }
+          if (closest == null || dist < closest.dist) {
+            //closest = this.features[i];
+            closest = this.filtered[i];
+            closest.dist = dist;
+          }
         }
-
-        if (closest == null || dist < closest.dist) {
-          //closest = this.features[i];
-          closest = this.filtered[i];
-          closest.dist = dist;
+        return closest;
+      },
+      // Returns the index of the first feature of a given milestone string.
+      firstOfMilestone: function(milestone, opt_startFrom) {
+        var start = opt_startFrom != undefined ? opt_startFrom : 0;
+        for (var i = start, feature; feature = this.filtered[i]; ++i) {
+          if (feature.first_of_milestone && [
```

Comment:
Doing a `==` comparison and relying on implicit type coercion seems like a lesser evil than creating new arrays and explicitly converting everything to `String`:

    if (feature.first_of_milestone && (milestone == feature.shipped_milestone || milestone == feature.impl_status_chrome)) {
      return i;
    }